pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -11,43 +11,37 @@ import (
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
-// We use pointers in a spec field when the field is optional, to allow differentiate
-// Between an unset value from the zero value of the field.
-// This is a common convention
-// used in kubernetes: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#optional-vs-required
+// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 
 // APIManagerSpec defines the desired state of APIManager
+// +k8s:openapi-gen=true
 type APIManagerSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-
 	APIManagerCommonSpec `json:",inline"`
 	// +optional
-	ApicastSpec *ApicastSpec `json:"apicast,omitempty"`
+	Apicast *ApicastSpec `json:"apicast,omitempty"`
 	// +optional
-	BackendSpec *BackendSpec `json:"backend,omitempty"`
+	Backend *BackendSpec `json:"backend,omitempty"`
 	// +optional
-	SystemSpec *SystemSpec `json:"system,omitempty"`
+	System *SystemSpec `json:"system,omitempty"`
 	// +optional
-	ZyncSpec *ZyncSpec `json:"zync,omitempty"`
+	Zync *ZyncSpec `json:"zync,omitempty"`
 	// +optional
-	WildcardRouterSpec *WildcardRouterSpec `json:"wildcardRouter,omitempty"`
+	WildcardRouter *WildcardRouterSpec `json:"wildcardRouter,omitempty"`
 	// +optional
-	HighAvailabilitySpec *HighAvailabilitySpec `json:"highAvailability,omitempty"`
+	HighAvailability *HighAvailabilitySpec `json:"highAvailability,omitempty"`
 }
 
 // APIManagerStatus defines the observed state of APIManager
+// +k8s:openapi-gen=true
 type APIManagerStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	Conditions []APIManagerCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,4,rep,name=conditions"`
+	Conditions []APIManagerCondition `json:"conditions,omitempty" protobuf:"bytes,4,rep,name=conditions"`
```

Comment:
`patchStrategy` and `pathMerge` tags removed. `operator-sdk generate openapi` failed

Still unknown why